tests/rotation.py

```diff
@@ -28,19 +28,23 @@ def run(args):
     ) as network:
         network.start_and_join(args)
 
+        iterations = 10 if args.long_tests else 3
```

Comment:
If this is the only thing long_tests does, I think we may as well specify ROTATION_ITERATIONS directly? Just a thought.

Reply:
I feel like we may want to keep this option generic enough to be used by other tests too? I'm thinking if we aim to keep the CI as quick as possible and only run longer tests to the Daily, this may be a pattern that we use more.